guides/plugins/apps/storefront/cookies-with-apps.md

```diff
@@ -0,0 +1,99 @@
+
+## Create a cookie group
+
+When adding multiple cookies through your app it may become in handy to group them. This makes it possible for the customer to accept all of your cookies at once and additionally enhances the readability of the cookie consent manager.
+
+To add a cookie group you can simply add a `groups` section within your `cookies` section in your `manifest.xml`. In the following example we use the cookie that we created in the previous section but display it in a cookie group:
+
+{% code title="manifest.xml" %}
+```markup
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <name>ExampleAppWithCookies</name>
+        <version>1.0.0</version>
+        <!-- other meta data goes here -->
+    </meta>
+    <cookies>
+        <group>
+            <snippet-name>example-app-with-cookies.cookie-group.name</snippet-name>
+            <snippet-description>example-app-with-cookies.cookie-group.description</snippet-description>
+            <entries>
+                <cookie>
+                    <cookie>my-cookie</cookie>
+                    <snippet-name>example-app-with-cookies.my-cookie.name</snippet-name>
+                    <snippet-description>example-app-with-cookies.my-cookie.description</snippet-description>
+                    <value>a static value for the cookie</value>
+                    <expiration>1</expiration>
+                </cookie>
+            </entries>
+        </group>
+    </cookies>
+</manifest>
+```
+{% endcode %}
+
+A `group` element consists of three child elements to configure the cookie group. Here is a description of all of them:
+
+* `snippet-name` (required): A string that represents the label of the cookie group in the cookie consent manager. To provide translations this should be the key of a storefront snippet.
+* `entries` (required): Contains the grouped cookies. It is a collection of `cookie` elements described in the previous section. 
+* `snippet-description` (optional): A string that represents the description of the cookie group in the cookie consent manager. To provide translations this should be the key of a storefront snippet.
+
+For a complete reference of the structure of the manifest file, take a look at the [Manifest reference](../../../../resources/references/app-reference/manifest-reference.md).
+
+## Snippet handling
+
+As already mentioned in the previous sections both the `cookie` as well as the `group` elements can contain `snippet-name` and `snippet-description` child elements. Although their values can be strings that will be displayed in the storefront as is, the prefered way to set up cookie names and descriptions is to provide storefront snippets.
```

Comment:
Outline that it's prefered because, without a snippet, it wouldn't get translated.